templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="{% url 'taxi:index' %}">Home</a>
+  </li>
+  <li>
+    <a href="#">Driver</a>
+  </li>
+  <li>
+    <a href="#">Manufacturer</a>
```

Comment:
The link for 'Manufacturer' is currently a placeholder (`#`). If there is a specific URL or view for the Manufacturer section, consider replacing `#` with `{% url 'app_name:view_name' %}` where `app_name:view_name` corresponds to the actual view.